translate/bytecode_test.go

```diff
@@ -16,9 +16,127 @@
 
 package translate_test
 
-import "testing"
+import (
+	"bytes"
+	"testing"
 
-// TODO: implement test cases :-)
-func TestBytecode_Emit(t *testing.T) {
+	"github.com/DE-labtory/koa/opcode"
+	"github.com/DE-labtory/koa/translate"
+)
 
+func TestBytecode_Emerge(t *testing.T) {
+	tests := []struct {
+		op       opcode.Type
+		operands [][]byte
+		result   translate.Bytecode
+	}{
+		{
+			op: opcode.Add,
+			operands: [][]byte{
+				{0x01},
+				{0x02},
+			},
+			result: translate.Bytecode{
+				RawByte: []byte{byte(opcode.Add), 0x01, 0x02},
+				AsmCode: []string{"Add", "01", "02"},
+			},
+		},
+		{
+			op: opcode.Mul,
+			operands: [][]byte{
+				{0x11},
+				{0x22},
+			},
+			result: translate.Bytecode{
+				RawByte: []byte{byte(opcode.Mul), 0x11, 0x22},
+				AsmCode: []string{"Mul", "11", "22"},
+			},
+		},
+		{
+			op: opcode.Mload,
+			operands: [][]byte{
+				{0xff},
+				{0xff},
+			},
+			result: translate.Bytecode{
+				RawByte: []byte{byte(opcode.Mload), 0xff, 0xff},
+				AsmCode: []string{"Mload", "ff", "ff"},
+			},
+		},
+		// How to deal with byte overflow
+		{
```

Comment:
Does `Loadfunc` need only 1 operand?

Reply:
Yes I intentionally put 2 operands to show that num of operands is not check here

If we add checking logic to that func this tcs should be blocked